compiler/compile/OSRData.hpp

```diff
@@ -119,7 +119,7 @@ class TR_OSRCompilationData
                            int32_t symRefOrder, int32_t symSize, bool takesTwoSlots);
    void ensureSlotSharingInfoAt(const TR_ByteCodeInfo& T);
    void addInstruction(TR::Instruction* instr);
-   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo &bcInfo);
+   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo bcInfo);
```

Comment:
Yes we would. The external API to the rest of the compiler should not be a reference because bcInfo is modified in the iterative loop inside the implementation on line 139 of OSRData.cpp. The implementation internally is iterative so I don't think it makes sense to create an impl method that takes a reference which is internal. I believe the copy is required and is a necessary safety mechanism - otherwise the bcInfo in the caller context will be modified by the search being performed which is not the desired behavior and would be entirely surprising.